types/three/index.d.ts

```diff
@@ -3,7 +3,7 @@
 // Definitions by: Josh Ellis <https://github.com/joshuaellis>
 //                 Nathan Bierema <https://github.com/Methuselah96>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// Minimum TypeScript Version: 3.6
+// Minimum TypeScript Version: 3.8
```

Comment:
Why is this change necessary?

Reply:
Because doing `export * as MathUtils from 'PATH'` fails the test suite @ 3.6 & 3.7 for some reason.

Reply:
👍 Looks like it's due to `export * as` syntax being added in 3.8 (https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#export--as-ns-syntax).

Reply:
Looks like you would have to bump the minimum required version of `aframe` as well though. Maybe it's better to just stick with this for now?:
```
import * as MathUtils from './math/MathUtils';
export { MathUtils };
```

Reply:
Yeah, the issue is aframe min version is 3.6, I'm not sure the process to resolve this?

Reply:
Yeah just seen that recommended on Discord, will do that tonight